### DISCUSSION TITLE
rtmpdump versioning issue when running "brew cleanup"

### DISCUSSION THREAD

**Discussion by lycogno** [Homebrew/discussions#4768] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.9
ORIGIN: https://github.com/Homebrew/brew
HEAD: b0985dabec8517ab5afda7cfee7d59f1da4ed6ce
Last commit: 2 hours ago
Core tap JSON: 05 Sep 18:18 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 14.0.3 build 1403
Git: 2.39.2 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.88.1 => /usr/bin/curl
macOS: 13.4.1-arm64
CLT: 14.3.1.0.1.1683849156
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

When running `brew cleanup`, I get the following warning:
Warning: Skipping rtmpdump: most recent version 2.4-20151223_2 not installed

However, on running `brew upgrade rtmpdump`, the output is the following:
Warning: rtmpdump 2.4+20151223_2 already installed

This appears to be caused by a version number issue, and I have not observed any immediate crashes or bugs as a result. I'm confused on how to resolve this, any tips?

**Comment by p-linnane** [comment, marked as answer]
Please try `brew reinstall rtmpdump`.

If that doesn't work, try: `brew fetch rtmpdump && brew reinstall rtmpdump`

**Reply by lycogno** [reply]
Thanks, seems to be fine now.

**Comment by suportehdc** [comment]
The command **brew upgrade rtmpdump** works for me.
